data/distro.yml

```diff
@@ -140,7 +140,7 @@
   steps:
     - name: Install Flatpak
       text: "
-        <p>A flatpak package is available in Debian 10 (Buster) and newer. To install it, run the following as root:</p>
+        <p>A Flatpak package is available in Debian 10 (Buster) and newer. To install it, run the following as root:</p>
```

Comment:
Thanks, however I am not sure if this is correct considering it is a name of the "flatpak" package itself (that is usually lowercase).